src/main/java/games/strategy/engine/framework/startup/launcher/ServerLauncher.java

```diff
@@ -402,44 +402,44 @@ private static String getConnectionLostFileName() {
     return GameDataFileUtils.addExtension(
         "connection_lost_on_" + DateTimeFormatter.ofPattern("MMM_dd_'at'_HH_mm").format(LocalDateTime.now()));
   }
-}
-
-
-class ServerReady implements IServerReady {
-  private final CountDownLatch m_latch;
-  private final int m_clients;
 
-  ServerReady(final int waitCount) {
-    m_clients = waitCount;
-    m_latch = new CountDownLatch(m_clients);
-  }
+  static class ServerReady implements IServerReady {
```

Comment:
I'm not sure... I thought the same thing about reflection and it didn't work...
It't probably going to work, but I can't give a guarantee, as I'm currently unable to check on a test server because I can't get postresql working...